packages/tools/devtools/devtools-core/tsconfig.json

```diff
@@ -4,7 +4,7 @@
 		"rootDir": "src",
 		"outDir": "./lib",
 		"types": ["chai", "mocha"],
-		"noUncheckedIndexedAccess": false,
+		"noUncheckedIndexedAccess": true,
```

Comment:
I see in the PR for map we just removed the line for "false". That made me realize it's not set in the base node16 config, so does that other PR need to set it to true explicitly as well?

Reply:
I will remove it here as well, removing it will follow tsconfig.base.json, which has this set to true